unittest/test_load.py

```diff
@@ -79,6 +79,9 @@ def test_solo(self):
     def test_solo12(self):
         self.check('solo12', 19, 18)
 
+    def test_solo(self):
```

Comment:
I think the name is not correct here.

Reply:
Thanks for the quick review! I've corrected the name.